From d7dd61b81bfd9cbb4fcaf2009d39bf944a94422f Mon Sep 17 00:00:00 2001
From: Ryan QIAN <b10692@freescale.com>
Date: Wed, 12 Mar 2008 14:09:57 -0500
Subject: [PATCH 010/505] ENGR00030479 Update IMX37_3STACK exclude list

Add IMX37_3STACK to the exclude list of firi, mu, owire, pf and security

Signed-off-by: Ryan QIAN <b10692@freescale.com>
---
 test/mxc_firi_test/Makefile     | 4 ++--
 test/mxc_mu_test/Makefile       | 4 ++--
 test/mxc_owire_test/Makefile    | 4 ++--
 test/mxc_pf_test/Makefile       | 4 ++--
 test/mxc_security_test/Makefile | 8 ++++++--
 5 files changed, 14 insertions(+), 10 deletions(-)

diff --git a/test/mxc_firi_test/Makefile b/test/mxc_firi_test/Makefile
index dd99ba3..bb42ee8 100644
--- a/test/mxc_firi_test/Makefile
+++ b/test/mxc_firi_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="MXC27520EVB MXC27530EVB IMX27ADS MXC30020EVB MXC30031ADS"
+EXCLUDE_LIST:="MXC27520EVB MXC27530EVB IMX27ADS MXC30020EVB MXC30031ADS IMX37_3STACK"
 
 # list of platforms which do not have either SIR or FIR
 IR_EXCLUDE_LIST:=
@@ -7,7 +7,7 @@ IR_EXCLUDE_LIST:=
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/autorun-firi.sh
 else
-OBJS = 
+OBJS =
 endif
 
 ifeq (,$(findstring $(PLATFORM), $(IR_EXCLUDE_LIST)))
diff --git a/test/mxc_mu_test/Makefile b/test/mxc_mu_test/Makefile
index 05fb9ba..9f8acae 100644
--- a/test/mxc_mu_test/Makefile
+++ b/test/mxc_mu_test/Makefile
@@ -1,12 +1,12 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX27ADS IMX31ADS"
+EXCLUDE_LIST:="IMX27ADS IMX31ADS IMX37_3STACK"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = \
 	$(OBJDIR)/mxc_mu_test.out \
         $(OBJDIR)/autorun-mu.sh
 else
-OBJS = 
+OBJS =
 endif
 
 all : $(OBJS)
diff --git a/test/mxc_owire_test/Makefile b/test/mxc_owire_test/Makefile
index 2b34282..802ee65 100644
--- a/test/mxc_owire_test/Makefile
+++ b/test/mxc_owire_test/Makefile
@@ -1,10 +1,10 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:=IMX31_3STACK
+EXCLUDE_LIST:="IMX31_3STACK IMX37_3STACK"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/autorun-owire.sh
 else
-OBJS = 
+OBJS =
 endif
 
 all : $(OBJS)
diff --git a/test/mxc_pf_test/Makefile b/test/mxc_pf_test/Makefile
index ae2fd9f..9e893f2 100644
--- a/test/mxc_pf_test/Makefile
+++ b/test/mxc_pf_test/Makefile
@@ -1,10 +1,10 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:= IMX27ADS
+EXCLUDE_LIST:="IMX27ADS IMX37_3STACK"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/autorun-pf.sh $(OBJDIR)/mxc_pf_test.out
 else
-OBJS = 
+OBJS =
 endif
 
 all : $(OBJS)
diff --git a/test/mxc_security_test/Makefile b/test/mxc_security_test/Makefile
index d3db7cb..24e27eb 100644
--- a/test/mxc_security_test/Makefile
+++ b/test/mxc_security_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:= 
+EXCLUDE_LIST:=IMX37_3STACK
 
 # list of platforms that have hacc
 HACC_LIST:=MXC91131EVB
@@ -7,7 +7,7 @@ HACC_LIST:=MXC91131EVB
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/mxc_security_test.out
 else
-OBJS = 
+OBJS =
 endif
 
 ifeq ($(PLATFORM),$(findstring $(PLATFORM),$(HACC_LIST)))
@@ -16,6 +16,10 @@ else
 SCRIPTS = $(OBJDIR)/autorun-rtic.sh
 endif
 
+ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
+SCRIPTS =
+endif
+
 all : $(OBJS) $(SCRIPTS)
 
 .PHONY: clean
-- 
1.8.0

